dockers/docker-ptf/Dockerfile.j2

```diff
@@ -96,8 +96,11 @@ RUN mkdir /var/run/sshd \
  && sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config \
  && sed -i '$aUseDNS no' /etc/ssh/sshd_config
 
-COPY ["supervisord.conf", "sshd.conf", "ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+COPY ["supervisord.conf", "/etc/supervisor/"]
+COPY ["conf.d/supervisord.conf", "conf.d/sshd.conf", "conf.d/ptf_nn_agent.conf", "/etc/supervisor/conf.d/"]
+
+RUN mkdir -p /var/log/supervisor
 
 EXPOSE 22
 
-ENTRYPOINT ["/usr/bin/supervisord"]
+ENTRYPOINT ["/usr/local/bin/supervisord", "-c", "/etc/supervisor/supervisord.conf"]
```

Comment:
Otherwise it gives following in the log
/usr/local/lib/python2.7/dist-packages/supervisor/options.py:298: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
  'Supervisord is running as root and it is searching '

So I provide -c to suppress this message.